src/schematic/traits.rs

```diff
@@ -29,10 +29,10 @@ pub const INGRESS: &str = "ingress";
 pub const AUTOSCALER: &str = "auto-scaler";
 pub const MANUAL_SCALER: &str = "manual-scaler";
 pub const VOLUME_MOUNTER: &str = "volume-mounter";
-pub const INGRESS_V1ALPHA1: &str = "ingress.core.oam.dev/v1alpha1";
-pub const AUTOSCALER_V1ALPHA1: &str = "auto-scaler.core.oam.dev/v1alpha1";
-pub const MANUAL_SCALER_V1ALPHA1: &str = "manual-scaler.core.oam.dev/v1alpha1";
-pub const VOLUME_MOUNTER_V1ALPHA1: &str = "volume-mounter.core.oam.dev/v1alpha1";
+pub const INGRESS_V1ALPHA1: &str = "ingress";
```

Comment:
We already have [INGRESS](https://github.com/oam-dev/rudr/pull/516/files#diff-7a54cab906170550b35ca377ea3dc62eR28) here, why don't remove it directly ?

Reply:
Remove old remnants instead.